examples/resource-examples.yaml

```diff
@@ -0,0 +1,42 @@
+# Example: Resources
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - farm: https://farm.example/
+    ose: https://www.opensourceecology.org
+
+'@id': rgh:valueflows/valueflows/master/examples/resource-examples.yaml
+'@graph':
+
+  - '@id': ose:822b6baf-d66f-45a0-98f2-57450e2aec23
+    '@type': ResourceSpecification
+    name: Tractor - LifeTrac 6
+    url: https://www.opensourceecology.org/portfolio/tractor/
+    image: https://www.opensourceecology.org/wp-content/uploads/2014/02/Tractor_gray.png
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q39495
+    skos:note: A versatile, 4-wheel drive, hydraulically-driven tractor with 18 to 200 horsepower capacity for agriculture, construction and other utility duties.
+
+  - '@id': farm:fbff9852-36ca-4364-a943-bc0b49e1cab5
+    '@type': EconomicResource
+    name: Tractor - LifeTrac 6
+    trackingIdentifier: 889jcd00s # the serial number assigned by the manufacturer
+    resourceConformsTo: ose:822b6baf-d66f-45a0-98f2-57450e2aec23
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': farm:07819789-dd51-44c3-b35c-9210165bc832
+    '@type': EconomicResource
+    name: 12 kg case organic carrots
```

Comment:
can we start without cases, just 600kg of carrots and later we can add variant which expresses 12kg cases in machine readable way

Reply:
done